BLSQ/iaso
# IA-2239: improve setup

Currently, if you setup an admin account on iaso, you will get this as homepage


The next step is to connect to /api/setupaccount/ and setup your first account. 

We should replace the content of the page above by a form with the same content as `/api/setupaccount/` and a logout button that allows to switch to the newly created account when it’s done. 

This account setup should also automatically create:
- a data source
- a project
Also only superuser or staff user without `iaso_profile` should be able to see account setup page

Related JIRA tickets : IA-2239

## Self proofreading checklist


## Changes

- Allow user without `iaso_profile` to access the react application (routes, redirections, ...)
- new domain for setup 
- setup page with 3 different views: 
     1. Not admin (superuser or staff)
     2. Admin view with account setup form
     3. Success message         
- added logout button to topbar
- allow user to switch the language without `iaso_profile`
- change api validation message in order to display translated message in the front
- update bluesquare-component to use the correct icon on password field

## How to test

Create three users without `iaso_profile`: staff, superuser and a normal one
Access to the dashboard with those profiles.
Normal user should only see a message saying you should contact an admin
Both staff and superuser should be able to create an account and see the success message
Try also to switch the language.
Try to create an account or a user with an already existing account name or user name.

## Print screen / video
<img width="1673" alt="Screenshot 2023-08-29 at 11 18 19" src="https://github.com/BLSQ/iaso/assets/12494624/797d67c4-b155-460a-8ba9-dd55341d6dc0">
<img width="1679" alt="Screenshot 2023-08-29 at 11 17 45" src="https://github.com/BLSQ/iaso/assets/12494624/0f7ffa64-b7e5-43d8-9ee9-e4cd6a641a88">
<img width="1680" alt="Screenshot 2023-08-29 at 11 17 33" src="https://github.com/BLSQ/iaso/assets/12494624/42660f5e-4a6f-4c3c-8eda-58d248d45ae1">
<img width="1674" alt="Screenshot 2023-08-29 at 11 16 42" src="https://github.com/BLSQ/iaso/assets/12494624/1edb6814-7625-413b-a21d-b15d668eac27">
